src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -74,19 +71,21 @@ private String getDefendantResponseIssuedEmailTemplate(Claim claim) {
         Party party = claim.getClaimData().getClaimant();
         Response response = claim.getResponse().orElseThrow(() -> new IllegalStateException("Response expected"));
 
-        if (isFullDefenceAndNoMediation(response)) {
+        if (isFullDefenceAndDispute(response)) {
             return getEmailTemplates().getDefendantResponseWithNoMediationIssued();
-        }
-
-        if (party instanceof Individual || party instanceof SoleTrader) {
+        } else if (party instanceof Individual || party instanceof SoleTrader
+            || party instanceof Company || party instanceof Organisation) {
             return getEmailTemplates().getDefendantResponseIssuedToIndividual();
-        } else if (party instanceof Company || party instanceof Organisation) {
-            return getEmailTemplates().getDefendantResponseIssuedToCompany();
         } else {
             throw new NotificationException(("Unknown claimant type " + party));
         }
     }
 
+    private boolean isFullDefenceAndDispute(Response response) {
```

Comment:
Please move the utility methods in class `uk.gov.hmcts.cmc.domain.utils.ResponseUtils` and add unit test

Reply:
@sabahirfan I'll do this, but I will also have to duplicate the method `isFullDefenceAndNoMediation` as it uses it. It would be better if it was moved into ResponseUtils too, however it's used in 3 other places currently. 
Do you want me to move `isFullDefenceAndNoMediation` into ResponseUtils too and change their calls to use it in that file?

Reply:
yes please.

Reply:
Will do! 👍